components/page.js

```diff
@@ -60,31 +60,43 @@ export default ({ hackathons, currentPath, children }) => {
   return (
     <Container>
       <Sidebar currentPath={currentPath} hackathons={hackathons} />
-      <div style={{ 'max-width': '80vw' }}>
+      <div style={{ width: '80vw' }}>
         <HeaderContainer>
           <Header>{currentPath}</Header>
           {loading && <LoadingImage src={LoadingGif} />}
         </HeaderContainer>
         <HackathonNavBar>
           <Link href="/[id]/intro" as={`/${currentPath}/intro`}>
             <NavItem
-              onClick={() => setLoading(true)}
+              onClick={() => {
+                if (!window.location.href.includes('intro')) {
```

Comment:
Small suggestion but can Next's router be used here to get the current pathname? Something like `const router = useRouter()` then 
```suggestion
                if (!router.pathname.includes('intro')) {
```

Reply:
it can but it comes on the reliance of extra code which I'm not a fan of. Since we can use window (because our app is not server side) we might as well use it.